.travis.yml

```diff
@@ -4,8 +4,14 @@ before_install:
   - wget http://www.builtwithmonogame.com/releases/v3.4/MonoGame.Linux.zip
```

Comment:
I have no Idea what this travis.yml stuff does, I trust that you did this well ^^.

Reply:
Yes. It's building after all xD